Project.toml

```diff
@@ -34,7 +34,7 @@ Boltz = "1"
 ChainRulesCore = "1"
 ComponentArrays = "0.15.17"
 ConcreteStructs = "0.2"
-DataInterpolations = "5, 6"
```

Comment:
don't remove 5

Reply:
That causes conflict too, with Symbolics

Reply:
Just saw your comment on the compat PR in Boltz, I'll find a workaround

Reply:
Why would 5 be needed?

Reply:
Fixed